components/Form/NewContactForm.js

```diff
@@ -0,0 +1,285 @@
+        address: "",
+      },
+    ],
+  });
+
+  const handleInputChange = (event, data) => {
+    const { name, value } = event.currentTarget;
+
+    if (name === "profileIconSource") {
+      setFormData((prevFormData) => ({
+        ...prevFormData,
+        [name]: data,
+      }));
+    } else if (name === "hobbies") {
+      setFormData((prevFormData) => ({
+        ...prevFormData,
+        [name]: value.split(","),
+      }));
+    } else {
+      setFormData((prevFormData) => ({
+        ...prevFormData,
+        [name]: value,
+      }));
+    }
+  };
+
+  const handleContactOptionChange = (event) => {
+    const { name, value } = event.currentTarget;
+    const { contactOptions } = formData;
+    const index = contactOptions.findIndex(
+      (contactOption) => contactOption.type === name
+    );
+
+    if (numberTypes.includes(contactOptions[index].type)) {
+      contactOptions[index].number = value;
+    } else {
+      contactOptions[index].address = value;
+    }
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      contactOptions,
+    }));
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (!formData.profileIconSource) {
+      alert("Please select a profile picture");
+      return;
+    }
+    addContact(formData);
+    router.push("/contacts");
+  };
+
+  return (
+    <>
+      <StyledNewContactForm onSubmit={handleSubmit}>
+        <label htmlFor="profileIconSource"></label>
+        <ProfilePictureSelect
+          selectedPicture={formData.profileIconSource}
+          handlePictureSelect={handleInputChange}
+        />
+        <h3>Personal Information</h3>
+        <label htmlFor="nickname">Nickname:</label>
+        <StyledInput
+          type="text"
+          id="nickname"
+          name="nickname"
+          value={formData.nickname}
+          onChange={handleInputChange}
+          required
+        />
+
+        <label htmlFor="first_name">First Name:</label>
+        <StyledInput
+          type="text"
+          id="first_name"
+          name="first_name"
+          value={formData.first_name}
+          onChange={handleInputChange}
+          required
+        />
+
+        <label htmlFor="last_name">Last Name:</label>
+        <StyledInput
+          type="text"
+          id="last_name"
+          name="last_name"
+          value={formData.last_name}
+          onChange={handleInputChange}
+        />
+
+        <label htmlFor="birthday">Birthday:</label>
+        <StyledInput
+          type="date"
+          id="birthday"
+          name="birthday"
+          value={formData.birthday}
+          onChange={handleInputChange}
+        />
+
+        <label htmlFor="friendship_status">Friendship Status:</label>
+        <StyledSelect
+          id="friendship_status"
+          name="friendship_status"
+          value={formData.friendship_status}
+          onChange={handleInputChange}
+        >
+          <option value="set Status">Set status</option>
```

Comment:
Pro tip: You can set the "label" option that is not really meant to be selected to `disabled` to prevent it from being selectable in the dropdown.

 ```suggestion
           <option value="set Status" disabled>Set status</option>
```